packages/comms/package.json

```diff
@@ -69,7 +69,7 @@
         "tmp": "0.2.1"
     },
     "devDependencies": {
-        "@hpcc-js/bundle": "^2.11.1",
+        "@hpcc-js/bundle": "^2.10.23",
```

Comment:
Yes, I forgot to change that back. I was getting an error for that when installing dependencies, probably because I had done so from within the comms package. But I had looked at https://www.npmjs.com/package/@hpcc-js/bundle and didn't see that version listed. I didn't think to check the version in the actual package.